torch/csrc/jit/codegen/cuda/fusion_segmenter.h

```diff
@@ -0,0 +1,365 @@
+#pragma once
+
+#include <torch/csrc/jit/codegen/cuda/fusion.h>
+#include <torch/csrc/jit/codegen/cuda/ir_base_nodes.h>
+#include <torch/csrc/jit/codegen/cuda/kernel_cache.h>
+#include <torch/csrc/jit/codegen/cuda/scheduler.h>
+#include <torch/csrc/jit/codegen/cuda/scheduler_registry.h>
+
+#include <deque>
+#include <list>
+#include <unordered_set>
+#include <vector>
+
+namespace torch {
+namespace jit {
+namespace fuser {
+namespace cuda {
+
+//  FusionSegmentFinder
```

Comment:
this would make a great doxygen comment (just replace `//` with `//!`  and move it to prefix the right class)

Reply:
Use this comment for `SegmentCandidateFinder`, thanks!